packages/mdc-menu/foundation.js

```diff
@@ -188,13 +199,24 @@ class MDCMenuFoundation extends MDCFoundation {
     this.anchorMargin_.left = typeof margin.left === 'number' ? margin.left : 0;
   }
 
+  /** @param {boolean} rememberSelection */
+  setRememberSelection(rememberSelection) {
+    this.rememberSelection_ = rememberSelection;
+    this.setSelectedIndex(-1);
+  }
+
   /**
    * @param {?number} focusIndex
    * @private
    */
   focusOnOpen_(focusIndex) {
     if (focusIndex === null) {
-      // First, try focusing the menu.
+      // First focus the last selected item if we're remembering selections and it has been set
```

Comment:
flip this
"If this instance of MDCMenu remembers selections, and the user has made a selection, then focus the last selected item"